### ISSUE TITLE
Actions generator inserts test & route for already existing actions

### ISSUE THREAD

**Issue by stanislas-m** [gobuffalo/buffalo#412]
```bash
$ buffalo g a forums index
Buffalo version 0.8.1.2

--> [warning] skipping ForumsIndex since it already exists
--> actions/forums.go
--> actions/forums_test.go
--> goimports -w .
```

actions/forums.go is ok, but forums_test.go isn't:

``` go
func (as *ActionSuite) Test_Forums_Index() {
	as.Fail("Not Implemented!")
}

func (as *ActionSuite) Test_Forums_Index() {
	as.Fail("Not Implemented!")
}
```

The route shouldn't be inserted in app.go too:

``` go
app.GET("/forums/index", ForumsIndex)
app.GET("/forums/index", ForumsIndex)
```

**Comment by apaganobeleno** [comment]
@stanislas-m would you want to work on this one ? (i would like to take it 😁)

**Comment by stanislas-m** [comment]
@apaganobeleno Please help yourself! :)

**Comment by markbates** [comment]
This should be fixed regardless of 0.9.0.

**Comment by apaganobeleno** [comment]
Fixed by #416
